### ISSUE TITLE
Huge forward packet fan-out

### ISSUE THREAD

**Issue by behrad** [mcollina/ascoltatori#148]
It seems it can block the loop when sending wildcard messages to huge list of subscribers: https://github.com/mcollina/ascoltatori/blob/master/lib/trie_ascoltatore.js#L53-L55

I noticed usage of `fastparallel` in https://github.com/mcollina/mqemitter/blob/master/mqemitter.js, Can we improve here by `asyncjs`?

**Comment by mcollina** [comment]
Yes! Send a PR and I will inclde it in v3. For an async replacement, use
steed (it wraps fastparallel and others).
Il giorno dom 8 mag 2016 alle 09:27 Behrad <notifications@github.com> ha
scritto:
